src/main/java/io/seqera/tower/cli/commands/labels/ListLabelsCmd.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.commands.labels;
+
+import io.seqera.tower.ApiException;
+import io.seqera.tower.cli.commands.computeenvs.AbstractComputeEnvCmd;
+import io.seqera.tower.cli.commands.computeenvs.primary.GetCmd;
+import io.seqera.tower.cli.commands.computeenvs.primary.SetCmd;
+import io.seqera.tower.cli.exceptions.ShowUsageException;
+import io.seqera.tower.cli.responses.Response;
+import picocli.CommandLine;
+
+import java.io.IOException;
+
+@CommandLine.Command(
+        name = "list",
+        description = "List labels"
+)
+public class ListLabelsCmd extends AbstractComputeEnvCmd {
```

Comment:
All other subcommands do not have the parent command nomenclature at the name. So here would be `ListCmd` instead of `ListLabelsCmd`. I do not have a strong opinion on what is better, just right now all the others are like that.